src/intentManager.js

```diff
@@ -153,6 +154,33 @@ define([
         }
     }
 
+    function printTemplate(mug, options) {
+        var widget = widgets.media(mug, options),
+            uploadComplete = widget.handleUploadComplete,
+            field_regex = /{{\s*(.+)\s*}}/gm;
```

Comment:
> However, this still has problems. It will not allow a } character inside a {{ ... }} delimited range, which I assume should be legal given the double delimiter. Making a parser that would allow that is not trivial (maybe not even possible?) with a single regular expression.

Yeah that should probably be legal but I don't think that's really necessary to implement right now. I think this is mostly a one off for one domain. Will let @amstone326 or @ctsims comment on that though

> This seems like a good candidate to extract into a helper function and add some tests to verify it does what you expect.

Originally didn't do this because intentmanager doesn't actually return anything. However that's a silly way to think. Updated with tests